web3/gas_strategies/rpc.py

```diff
@@ -0,0 +1,11 @@
+"""
+TODO: Provide explanation of gas price engines
+"""
+
+
+def rpc_gas_pricing_strategy(web3, transaction_params=None):  # TODO: call eth.gasPrice
+    """
+    This is the default gas pricing strategy. It derives it's value from the eth_gasPrice
+    JSON-RPC call.
+    """
+    return web3.eth.gasPrice
```

Comment:
I'd like to make a small change.

* Change this function body to call directly out to the JSON-RPC endpoint

```python
def rpc_gas_pricing_strategy(...):
    web3.manager.request_blocking("eth_gasPrice", [])
```

Then, we can replace the current code in `web3.eth.gasPrice` with code that delegates to `self.web3.getGasPrice()`.  This gives us one definitive place to query gas pricing.

Reply:
Good thinking 😃  changed.